src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.Git.cs

```diff
@@ -190,21 +193,51 @@ private static Branch SwitchToBranchAndUpdate(string branchToSwitchTo, string ba
             return Enlistment.Head;
         }
 
-        private static void CreateDummyCommit(CancellationToken cancellationToken)
+        private static async Task<GitPullRequest> CreatePlaceholderBranchAsync(CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
-            var message = $"DUMMY INSERTION FOR {Options.InsertionName}";
-            var options = new CommitOptions()
+            var gitClient = ProjectCollection.GetClient<GitHttpClient>();
+            var repository = await gitClient.GetRepositoryAsync(project: Options.TFSProjectName, repositoryId: "VS", cancellationToken: cancellationToken);
+
+            var refs = await gitClient.GetRefsAsync(
+                repository.Id,
+                filter: $"heads/{Options.VisualStudioBranchName}",
+                cancellationToken: cancellationToken);
+            GitRef sourceBranch = refs.Single();
+
+            var branchName = GetNewBranchName();
+
+            _ = await gitClient.CreatePushAsync(new GitPush()
             {
-                AllowEmptyCommit = true
-            };
-            var watch = Stopwatch.StartNew();
-            var commit = Enlistment.Commit(
-                message,
-                InsertionToolSignature,
-                InsertionToolSignature,
-                options);
-            Console.WriteLine($"Committing took {watch.Elapsed.TotalSeconds} seconds");
+                RefUpdates = new[] {
+                    new GitRefUpdate()
+                    {
+                        Name = $"refs/heads/{branchName}",
+                        OldObjectId = sourceBranch.ObjectId,
+                    }
+                },
+                Commits = new[] {
+                    new GitCommitRef()
+                    {
+                        Comment = $"PLACEHOLDER INSERTION FOR {Options.InsertionName}",
+                        Changes = new GitChange[]
+                        {
+                            new GitChange()
+                            {
+                                ChangeType = VersionControlChangeType.Delete,
+                                Item = new GitItem() { Path = "/Init.ps1" }
+                            },
+                        }
+                    },
+                },
+            }, repository.Id, cancellationToken: cancellationToken);
```

Comment:
nit: consider removing the argument label

Reply:
Can't: there's an intermediate optional argument that I don't want to provide a value for.